app/src/main/java/com/example/mymovieinfo/dto/Specimen.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.mymovieinfo.dto
 
-class Specimen(var movieTitle: String = "",
+data class Specimen(var movieTitle: String = "",
                var movieRank: Int = 0,
```

Comment:
Declared 'Specimen' as a data class to tell compiler class is for holding data

Reply:
Good catch again.